rootfs/api/tests/test_release.py

```diff
@@ -151,12 +151,12 @@ def test_release_rollback(self):
             HTTP_AUTHORIZATION='token {}'.format(self.token))
         self.assertEqual(response.status_code, 201)
         # update the build to roll a new release
-        url = '/v2/apps/{app_id}/builds'.format(**locals())
-        body = {'image': 'autotest/example'}
-        response = self.client.post(
-            url, json.dumps(body), content_type='application/json',
-            HTTP_AUTHORIZATION='token {}'.format(self.token))
-        self.assertEqual(response.status_code, 201)
+        # url = '/v2/apps/{app_id}/builds'.format(**locals())
```

Comment:
If this code needs to remain commented out, let's add a `// TODO:` or `// NOTE:` comment above it explaining why.